packages/sys-apps/dstat/dstat-0.7.4-r1.exheres-0

```diff
@@ -25,8 +25,20 @@ DEPENDENCIES="
 
 RESTRICT="test"
 
+DEFAULT_SRC_PREPARE_PATCHES=(
+    "${FILES}"/Dool-fix-for-some-Python-deprecations.patch
+    "${FILES}"/Make-CSV-output-work-again.patch
+)
+
 DEFAULT_SRC_INSTALL_PARAMS=(
     DESTDIR="${IMAGE}"
     bindir="/usr/$(exhost --target)/bin"
 )
 
+src_prepare() {
+    default
+    edo sed \
+        -e "1 s|bin.\+$|$(exhost --target)/bin/python$(python_get_abi)|" \
```

Comment:
It is `/usr/bin/env python` out the box which runs eclectic-selected version — not what user expects when choosing `python_abis`. Especially as it uses additional dep on `six`.